health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/XlsPreviewNew.js

```
@@ -0,0 +1,69 @@
+import { PopUp, SVG, DownloadIcon, Button } from "@egovernments/digit-ui-react-components";
+import React from "react";
+import DocViewer, { DocViewerRenderers } from "@cyntler/react-doc-viewer";
+import { useTranslation } from "react-i18next";
+import { PRIMARY_COLOR } from "../utils";
+
+const ArrowBack = ({ className = "", height = "15", width = "15", styles = {} }) => {
+  return (
+    <svg className={className} style={styles} width={width} height={height} viewBox="0 0 15 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)